src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,42 @@
+@import "../../style/mixin.scss";
+@import "../../style/variables.scss";
+
+main {
+  width: 100vw;
+  height: 100vh;
+  @include flexCenter;
+  .introduce {
+    width: 500px;
+    height: 500px;
+    @include flexCenter;
+    flex-direction: column;
+    border: 3px solid;
+    border-radius: 10px;
+    padding: 10px;
+    .name {
+      @include textStyle(25px, bolder);
```

```suggestion
  height: 100vh;
  @include flexCenter;
  
  .introduce {
      @include flexCenter;
    width: 500px;
    height: 500px;
    flex-direction: column;
    border: 3px solid;
    border-radius: 10px;
    padding: 10px;
    
    .name {
      @include textStyle(25px, bolder);
```

선택자끼리 가독성을 위해 한칸씩 띄워주시고,
@include 를 사용할 때는 선택자 블록 최상위에 위치하도록 하는 것이 컨벤션입니다!